src/mount/mastercomm.cc

```diff
@@ -216,13 +212,32 @@ static inline void setDisconnect(bool value) {
 void fs_inc_acnt(uint32_t inode) {
 	std::unique_lock<std::mutex> acquiredFileLock(acquiredFileMutex);
 	acquiredFiles[inode]++;
+#ifdef _WIN32
+	if (gCleanAcquiredFilesPeriod > 0) {
+		if (acquiredFiles[inode] == 1) {
+			addAcquiredFileLastTimeUsed(inode, time(nullptr));
```

Comment:
Consider unifying this logic in the same function and avoid the `if - else` here.
IMHO, to handle this logic is responsibility of the function in charge of insert/update acquired files and not from `fs_inc_acnt`.

Reply:
Done